### DISCUSSION TITLE
How to remove https://localhost domin in production sitemap and robots.txt

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [lumeland/lume#349] [Q&A] [answered]
Today I successfully deployed my blog site to [the production domain](https://officialrajdeepsingh.dev/). I'm also config the domain name in lume.
```javascript
const site = lume(
  {
    server: {
      location: new URL("https://officialrajdeepsingh.dev"),
      page404: "/404/",
    },
  },
  { markdown },
);
```
But in production, `sitemap.xml`  and `robots.txt` both use http://localhost URL. Check out my [sitemap.xml file on production](https://officialrajdeepsingh.dev/sitemap.xml).

```xml
<loc>http://localhost/5-best-javascript-library-for-you.html</loc>
<lastmod>2020-11-08T10:57:05.000Z</lastmod>
```
But I can't find a solution yet.

**Comment by oscarotero** [comment, marked as answer]
The `location` option is not a server option, it must be moved to the root of the config:

```js
const site = lume(
  {
    location: new URL("https://officialrajdeepsingh.dev"),
    server: {
      page404: "/404/",
    },
  },
  { markdown },
);
```

**Reply by officialrajdeepsingh** [reply]
Thank you
